scripts/menu/server/sv_player_modal.lua

```diff
@@ -68,9 +68,25 @@ RegisterNetEvent('txAdmin:menu:spectatePlayer', function(id)
     if not target then
       return
     end
+    local tgtBucket = GetPlayerRoutingBucket(id)
+    local srcBucket = GetPlayerRoutingBucket(src)
+    if tgtBucket ~= srcBucket then 
+      ADMIN_DATA[tostring(src)].bucket = srcBucket
```

Comment:
I feel like this isn't the best place to store the data regarding bucket as we haven't traditionally used this map for non-admin related data but meh.

An alternative is we can store the original bucket on the calling client.